README.md

```diff
@@ -143,6 +143,11 @@ grunt.initConfig({
 });
 ```
 
+#### options.exclude
+Type: `Array`
+
+The specific dependencies packages to exclude from the compilation.
+
```

Comment:
You might include an example; the fact that it can take wild cards is non-obvious.